### PR TITLE
[DMS-21] Test and Improve the Release Workflow

### PR DESCRIPTION
**Pull request by andonyns**

### CODE REVIEW

File: .github/workflows/on-prerelease.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ on:
 
 env:
   ARTIFACTS_API_KEY: ${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}
-  AZURE_ARTIFACT_URL: "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
+  ARTIFACTS_FEED_URL: "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
   VSS_NUGET_EXTERNAL_FEED_ENDPOINTS: '{"endpointCredentials": [{"endpoint": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json","password": "${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}"}]}'
-  MANIFEST_FILE: "_manifest/spdx_2.2/manifest.spdx.json"
-  PACKAGE_NAME: "EdFi.DataManagementService"
+  MANIFEST_FILE: "manifest.spdx.json"
+  PACKAGE_NAME: "EdFi.DataManagementService.Api"
```
Comment (stephenfuqua) [inline .github/workflows/on-prerelease.yml:17]: Did we decide the package name does not need "Api" on it?

File: github-builder.psm1 [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+
+#requires -version 7
+
+$ErrorActionPreference = "Stop"
+
+function Get-VersionNumber {
```
[the remark this answers is not in the document]
Comment (stephenfuqua) [reply github-builder.psm1:11]: I sometimes get lax about this too. In fact, I was the one who wrote this code in another repository, so I will go ahead and use the word "lazy" now 😆. Let's be guided by the analyzer by adding some PowerShell-style documentation here.

**Comment by stephenfuqua** [comment]
The real test will come after we merge this to `main`.

**Review by stephenfuqua — APPROVED** [review]

**Review by simpat-adam — APPROVED** [review]

File: .github/workflows/on-prerelease.yml (hunk 1 above)
Comment (andonyns) [reply .github/workflows/on-prerelease.yml]: The reason why this was needed is that the `dotnet pack` is not reading the NuSpec configuration, will fix this in DMS-21-2

**Pull request merged**